install/kubernetes/helm/meshery/charts/meshery-consul/values.yaml

```diff
@@ -26,10 +26,10 @@ serviceAccount:
   # we need to split to every adapter with themself's cluster role. Although there a lot of work to do but can make the charts flexible.
 
   # Specifies whether a service account should be created
-  create: true
+  create: false
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
-  name: meshery-adapter-consul
+  name: meshery-server
```

Comment:
I have a consider here, please take a look on this 

https://github.com/meshery/meshery/blob/df44458404f7456b8d5e15434dcce13e912f06f2/install/kubernetes/helm/meshery/values.yaml#L34-L37

If the end-user replaces the default value of `service-account` from `meshery-server` to others that may cause the adapters can not work. right?

So, here we have a solution many people already use that is we set up the value on the top, we can set the value from the top chart to replace the `sub-charts` specific value. That can be good.

As you can see below, we can replace `fullnameOverride` for every `sub-charts`, and we can do the same for the `sa`.

https://github.com/meshery/meshery/blob/df44458404f7456b8d5e15434dcce13e912f06f2/install/kubernetes/helm/meshery/values.yaml#L93-L130


So, you can do it in the next PR, but please add the new relative issue for this.

Reply:
> I have a consider here, please take a look on this
> 
> https://github.com/meshery/meshery/blob/df44458404f7456b8d5e15434dcce13e912f06f2/install/kubernetes/helm/meshery/values.yaml#L34-L37
> 
> If the end-user replaces the default value of `service-account` from `meshery-server` to others that may cause the adapters can not work. right?
> 
> So, here we have a solution many people already use that is we set up the value on the top, we can set the value from the top chart to replace the `sub-charts` specific value. That can be good.
> 
> As you can see below, we can replace `fullnameOverride` for every `sub-charts`, and we can do the same for the `sa`.
> 
> https://github.com/meshery/meshery/blob/df44458404f7456b8d5e15434dcce13e912f06f2/install/kubernetes/helm/meshery/values.yaml#L93-L130
> 
> So, you can do it in the next PR, but please add the new relative issue for this.

+1 to what @Aisuko suggested